docs/sections/user_guide/index.rst

```diff
@@ -7,3 +7,5 @@ User Guide
    installation
    cli/index
    api/index
+   uw_yaml/index
```

Comment:
I'm not seeing this rendered in RtD, and am seeing errors when I run `make docs` on your branch.

Reply:
I think it's because there's no `uw_yaml/index.rst`.

Reply:
Oops!